lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -244,37 +258,43 @@ putSignedTransaction (tx, witnesses) = do
 putWitness :: TxWitness -> Put
 putWitness witness =
     case witness of
-        PublicKeyWitness xPub (Hash sig) -> do
+        PublicKeyWitness _xPub (Hash sig) -> do
             -- Witness sum type:
```

Comment:
we might want to define sum type that would describe the constants in this module, at least for documentation